README.md

```diff
@@ -28,6 +28,14 @@ Providers:
 
 ## Installation
 
+### Install via brew
+
+If you're macOS user and using [Homebrew](https://brew.sh/), you can install via brew command:
+
+```sh
+$ brew install terracognita
+```
+
```

Comment:
IMO I would move it below or above `Arch Linux` so then we have all the "platform" tools together.

Reply:
sounds good!